service/lib/agama/storage/finisher.rb

```diff
@@ -241,10 +241,30 @@ def label
         end
 
         def run
+          run_post_scripts
+          enable_init_scripts
+        end
+
+      private
+
+        # Run the post scripts
+        def run_post_scripts
```

Comment:
Not related to this PR, but I think all these things about scripts are misplaced in a storage finisher.

Reply:
Yes, there are quite some things that do not belong here. We will find better places as we continue moving stuff to Rust.